app/assets/stylesheets/widgets.scss

```diff
@@ -411,8 +411,9 @@
       font-size: var(--fs-base);
     }
     &.billboard {
-      max-height: calc(100vh - var(--header-height) - 2*var(--layout-padding));
-      overflow: auto;
```

Comment:
@benhalpern we [recently moved the overflow to the sidebar billboards](https://github.com/forem/forem/pull/19683/files). I've now removed it because its causing the bug with the dropdown, however, I wanted to check with you if there is a reason why you think we should have this max height and an overflow on the billboard in the sidebar? I guess we're trying to avoid the billboard scrolling more that the viewport height within mobile?

Reply:
@benhalpern a small nudge on this open question so we can clear this issue off the board~! Thanks!

Reply:
Bens response: 

The max height + overflow is necessary for smaller laptops or any situation where the billboard might go passed the height of the screen. Since the overall sidebar is fixed, the billboard needs to be scrollable to see the content in these cases. Generally billboard content is all visible in the sidebar, but this is to handle those edge cases elegantly.
So for that reason I don't think we can cleanly remove overflow like this.
In order to deal with the menu overflow problem, we could maybe either also have a "min height" for billboards of ~220px. This is sort of a hack, but almost all sidebar billboards will be taller than this anyway. The demo data is usually where this problem arises.
We could also dynamically remove the overflow property depending on the content of the billboard. I.e. observe the height of the billboard with JavaScript and remove the overflow property if the billboard is less than 220px.